libs/npm/core/src/hooks/api-editor/constants/Settings.ts

```diff
@@ -1,3 +1,12 @@
 export const Settings = {
   ApiPath: process.env.REACT_APP_API_URL ?? '/api',
+
+  // The follow keys are hardcoded setting names.
+  // Use them to fetch values from the settings lookup.
+  SubscriberUrl: 'SubscriberUrl',
```

Comment:
@jdtoombs @kylgarmor @fbarreta @CelloLiu 
Please note these keys for settings.  This will make identifying and maintain settings key constants easier.
It will also break the environment if I forget to update them.